docusaurus/docs/user-docs/settings/introduction.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introduction to General Settings
+description: Reviewing the audit logs in Strapi.
+sidebar_position: 2
+---
+
+The _Settings_ feature allows an administrator to adjust all ![Settings icon](/img/assets/icons/settings.svg) _Settings_ from the main navigation of the admin panel, which is divided into 4 sub sections:
+
+- _Global Settings_
+- _Administration panel_
+- _Email plugin_
+- _Users & Permissions plugin_
+
+<ThemedImage
+alt="Custom logo settings"
+sources={{
+    light: '/img/assets/settings/settings_custom-logo.png',
+    dark: '/img/assets/settings/settings_custom-logo_DARK.png',
+  }}
+/>
+
+## Global Settings
+
+The _Global Settings_ in Strapi allows administrators to configure logo in the _Overview & Custom Logo_ interface, manage _API Token_, manage _Media Library_, configure _Internationalization_ locales and configure _Single Sign-On_ and _Webhooks_.
+
+## Administration Panel
+
+The _Administration Panel_ Settings in Strapi allows administrators to configure _Roles_ and _Users_.
+
+### Roles
+
+The _Role_ interface allows administrators to edit the details of an administrator role and configure its permissions in detail. It can be accessed from _Administration panel_ > _Roles_ (see [Configuring Administrator Roles](/user-docs/users-roles-permissions/configuring-administrator-roles)).
+
+### Users
+
+The _Users_ interface allows administrators to edit the details of an end-users role and configure its permissions in detail. It can be accessed from _Administration panel_ > _Users_.
```

Comment:
I have the feeling that those paragraphs are very repetitive, and I'm not sure they're very necessary since neither are documented in the settings section anyways 🤔 
Perhaps we could just explain briefly what the "Administration Panel" section of settings contains and redirect users to the section of the user guide where everything is already documented?